code/datums/mutations/body.dm

```diff
@@ -230,6 +231,108 @@
 		return
 	owner.physiology.burn_mod *= 2
 
+/datum/mutation/human/badblink
+	name = "Spatial Instability"
+	desc = "Subject has a very weak link to spatial reality, and may be displaced. Often causes extreme nausea."
+	quality = NEGATIVE
+	text_gain_indication = "<span class='warning'>The space around you twists sickeningly.</span>"
+	text_lose_indication = "<span class'notice'>The space around you settles back to normal.</span>"
+	difficulty = 18//high so it's hard to unlock and abuse
+	instability = 10
+	var/warpchance = 0
+
+/datum/mutation/human/badblink/on_life()
+	if(prob(warpchance))
```

Comment:
Disable this if they are inside something else, like a cloning pod or cryo tube. There were issues with the Void mutation where someone developed it inside a cloning pod, ejected themselves early, but still had all the cloning pod traits attached to them.

Reply:
You sure do_teleport doesn't handle this already? Guess I'll test for it